src/TelemetryChannels/ServerTelemetryChannel/Shared/Implementation/TelemetryChannelEventSource.cs

```diff
@@ -30,6 +30,24 @@ public bool IsVerboseEnabled
             }
         }
 
+        [Event(5, Message = "Backend response {1} was not parsed. Some items may be dropped: {0}.", Level = EventLevel.Warning)]
+        public void BreezeResponseWasNotParsedWarning(string exception, string response, string appDomainName = "Incorrect")
+        {
+            this.WriteEvent(5, exception ?? string.Empty, response ?? string.Empty, this.ApplicationName);
```

Comment:
What do the "magic numbers" 5, 6, and 7 represent?  Are there constants for these with descriptive names?